src/modules/GroupChannel/context/GroupChannelProvider.tsx

```diff
@@ -32,8 +35,21 @@ import {
 } from '../../../utils/consts';
 import { useOnScrollPositionChangeDetectorWithRef } from '../../../hooks/useOnScrollReachedEndDetector';
 import { useMessageListScroll } from './hooks/useMessageListScroll';
+import { MessageListParams } from '../../Channel/context/ChannelProvider';
 
 type OnBeforeHandler<T> = (params: T) => T | Promise<T>;
+/**
+ * Because we depend on the Collection's default option (ChatSDK),
+ * we don't set following options in UIKit
+ */
+type GroupChannelMessageListParams = Omit<MessageListParams,
+  | 'reverse'
+  | 'includeMetaArray'
+  | 'includeReactions'
+  | 'includeThreadInfo'
+  | 'includeParentMessageInfo'
+  | 'showSubchannelMessagesOnly'
+>;
```

Comment:
We had decided to provide a separate migration guide for mapping. There's no need to maintain the interfaces provided by the previous UIKit. Let's follow the interface of the Chat SDK.

```ts
type GroupChannelMessageListParams = Omit<MessageCollectionParams, 'filter'> & MessageFilterParams;

// ...

function getCollectionCreator(groupChannel: GroupChannel, messageListQueryParams?: GroupChannelMessageListParams) {
  if (!messageListQueryParams) return undefined;

  return (defaultParams: MessageCollectionParams) => {
    return groupChannel.createMessageCollection({
      ...defaultParams,
      ...messageListQueryParams,
      filter: new MessageFilter(messageListQueryParams),
    });
  };
}
```